assets/javascripts/initializers/discourse-math-katex.js.es6

```diff
@@ -0,0 +1,60 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import loadScript from "discourse/lib/load-script";
+
+function ensureKaTeX() {
+  return loadScript("/plugins/discourse-math/katex/katex.min.js").then(() => {
```

Comment:
Sorry for probably a stupid question, but shouldn't these be included in the repo?

Reply:
Fixed. Feeling sorry that I forget to do `git add` on these files.